paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -734,3 +752,38 @@ def get_instances_from_ip(ip, instance_descriptions):
     :returns: list of instance description dicts"""
     instances = [instance for instance in instance_descriptions if instance['PrivateIpAddress'] == ip]
     return instances
+
+
+def get_autoscaling_info():
```

Comment:
Oh, except that creates a circular import because the autoscaling lib already imports `metastatus_lib` :-(

I think I'll have to leave this as is